src/NUnitFramework/tests/Assertions/AssertionHelperTests.cs

```diff
@@ -319,23 +319,23 @@ public void After()
         {
             var constraint = EqualTo(10).After(1000);
             Expect(constraint, TypeOf<DelayedConstraint>());
-            Expect(constraint.ToString(), EqualTo("<after 1000 <equal 10>>"));
+            Expect(constraint.ToString(), EqualTo("<after 1000 millisecond <equal 10>>"));
         }
 
         [Test]
         public void After_Property()
         {
             var constraint = Property("X").EqualTo(10).After(1000);
             Expect(constraint, TypeOf<DelayedConstraint>());
-            Expect(constraint.ToString(), EqualTo("<after 1000 <property X <equal 10>>>"));
+            Expect(constraint.ToString(), EqualTo("<after 1000 millisecond <property X <equal 10>>>"));
         }
 
         [Test]
         public void After_And()
         {
             var constraint = GreaterThan(0).And.LessThan(10).After(1000);
             Expect(constraint, TypeOf<DelayedConstraint>());
-            Expect(constraint.ToString(), EqualTo("<after 1000 <and <greaterthan 0> <lessthan 10>>>"));
+            Expect(constraint.ToString(), EqualTo("<after 1000 millisecond <and <greaterthan 0> <lessthan 10>>>"));
```

Comment:
I'm curious if additional intervals needs to be tested here? Since this looks like a test for the AssertionHelper and not related to After.